index.js

```diff
@@ -252,10 +253,20 @@ class ForwardEmail {
 
         const rcptTo = session.envelope.rcptTo.map(to => {
           return async () => {
-            const address = await this.getForwardingAddress(to.address);
+            let addresses = await this.getForwardingAddresses(to.address);
+            if (addresses.length > this.config.maxForwardedAddresses) {
+              if (this.config.maxForwardedAddresses > 0) {
+                addresses = addresses.slice(
+                  0,
+                  this.config.maxForwardedAddresses - 1
+                );
+              } else {
+                addresses = [];
```

Comment:
I think, this is a very special case - setting up a forwarding service without allowing any forwards. Maybe we should log this case, so it's easier for the service provider to understand why nothing gets forwarded? I think we should do the same for the case in which the maxForwardedAddresses are exceeded.